ui/src/components/DashboardInventory.test.js

```diff
@@ -0,0 +1,180 @@
+//@flow
+import React from 'react';
+import { screen } from '@testing-library/react';
+import { setupServer } from 'msw/node';
+import { rest } from 'msw';
+import DashboardInventory from './DashboardInventory';
+import {
+  waitForLoadingToFinish,
+  render,
+  FAKE_CONTROL_PLANE_IP,
+} from './__TEST__/util';
+import type { Alert } from '../services/alertUtils';
+import { useHighestSeverityAlerts } from '../containers/AlertProvider';
+import {
+  getNodesCountQuery,
+  getVolumesCountQuery,
+} from '../services/platformlibrary/k8s.js';
+
+const alertsCritical = [
+  {
+    id: 'alert1',
+    severity: 'critical',
+  },
+];
+const alertsWarning = [
+  {
+    id: 'alert2',
+    severity: 'warning',
+  },
+];
+const noAlerts = [];
+
+jest.mock('../services/platformlibrary/k8s.js', () => ({
+  __esModule: true, // Allows for the "default" import to work in the Mock injection
+  default: ({ children }) => <>{children}</>,
+  getNodesCountQuery: jest.fn(),
+  getVolumesCountQuery: jest.fn(),
+}));
+
+jest.mock('../containers/AlertProvider', () => ({
+  __esModule: true,
+  default: ({ children }) => <>{children}</>,
+  useHighestSeverityAlerts: jest.fn(),
+  useAlertLibrary: () => ({
+    getNodesAlertSelectors: () => {},
+    getVolumesAlertSelectors: () => {},
+  }),
+  highestAlertToStatus: (alerts?: Alert[]): string => {
+    return (alerts?.[0] && ((alerts[0].severity: any): string)) || 'healthy';
+  },
+}));
+
+const server = setupServer(
+  rest.get(
+    `http://${FAKE_CONTROL_PLANE_IP}:8443/api/prometheus/api/v1/query`,
+    (req, res, ctx) => {
+      const result = {
+        status: 'success',
+        data: {
+          resultType: 'vector',
+          result: [
+            {
+              metric: {
+                container: 'node-exporter',
+                device: '/dev/vdc',
+                endpoint: 'metrics',
+                fstype: 'xfs',
+                instance: '192.168.1.29:9100',
+                job: 'node-exporter',
+                mountpoint: '/mnt/testpart',
+                namespace: 'metalk8s-monitoring',
+                pod: 'prometheus-operator-prometheus-node-exporter-wk86s',
+                service: 'prometheus-operator-prometheus-node-exporter',
+              },
+              value: [1611905929.203, '96.86575443786982'],
+            },
+          ],
+        },
+      };
+      // return success status
+      return res(ctx.json(result));
+    },
+  ),
+);
```

Comment:
can you please explain what's the purpose to mock this? I don't see its used by the inventory card ;)